File-Formats/PDF/Convert-HTML-To-PDF/Windows-Forms.md

```diff
@@ -79,23 +75,23 @@ Create the btnCreate_Click event and add the following code in btnCreate_Click t
 
 // Initialize HTML to PDF converter.
```

Comment:
Continue numbering for create button and add code

Reply:
Added numbering for create button and add code